lib/commons/aria/index.js

```diff
@@ -1974,6 +1975,7 @@ lookupTable.role = {
 			'input:not([type])',
 			'textarea'
 		],
+		subclassRoles: ['searchbox'],
```

Comment:
IMO we should derive subclasses from the `superclassRole` that you've introduced, rather than duplicate this information.